test/consensus_cogateway/prove_consensus_gateway_proof.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
We have "-" in consensus-gateway directory name but here we are using "_".
We should align it.

Reply:
Done